frontend/src/metabase/query_builder/components/view/ViewHeader/components/HeaderBreadcrumbs/HeaderBreadcrumbs.jsx

```diff
@@ -40,22 +53,35 @@ export function HeadBreadcrumbs({
             {isValidElement(part) ? (
               part
             ) : (
-              <HeaderBadge
+              <Badge
+                className={cx(HeaderBreadcrumbsS.HeaderBadge, {
+                  [HeaderBreadcrumbsS.headVariant]: variant === "head",
+                })}
                 to={part.href}
                 icon={part.icon}
                 inactiveColor={badgeInactiveColor}
               >
                 {part.name}
-              </HeaderBadge>
+              </Badge>
```

Comment:
:x: The description should be visible when a question is opened, it's only visible after hover.
On mouseleave the description should disappear and the title should get vertically centered.

https://github.com/user-attachments/assets/e0a7a8ac-0290-46e3-95f7-059d3ed69a3e

Reply:
should be fixed 
https://github.com/user-attachments/assets/f23bbd88-0241-41a3-b71a-8fd84072a58b